wiki/Storyboard_Scripting/Commands/en.md

```diff
@@ -98,7 +98,7 @@ See the [shorthand](/wiki/Storyboard_Scripting/Shorthand) section for an explana
 
 | Affects | Value definition | Default value |
 | ------- | ---------------- | ------------- |
-| The location of the object in the play area. | An (x,y) position, as specified above. Decimals are not allowed. | An (x,y) position, as specified above. Decimals are not allowed. |
+| The location of the object in the play area. | An (x,y) position, as specified above. Decimals are accepted. | An (x,y) position, as specified above. Decimals are accepted. |
```

Comment:
for default value, it'd be the initial sprite/animation position

Reply:
see also #1660 I guess